bin/setup

```diff
@@ -1,8 +1,16 @@
 #!/usr/bin/env bash
 set -euo pipefail
 IFS=$'\n\t'
-set -vx
 
 bundle install
 
-# Do any other automated setup that you need to do here
+if [ ! -x "$(which minikube)" ]; then
+  echo -e "\n\033[0;33mIf you're going to run the tests, please follow the minikube setup instructions for your operating system:\nhttps://kubernetes.io/docs/getting-started-guides/minikube/#installation\033[0m"
+fi
+
+if [ ! -x "$(which kubectl)" ]; then
+  echo -e "\n\033[0;33mPlease install kubectl version 1.5.1 or higher:\nhttps://kubernetes.io/docs/user-guide/prereqs/\033[0m"
+else
+  KUBECTL_VERSION=$(kubectl version --short --client | grep -oe "v[[:digit:]\.]\+")
```

Comment:
👍 for the snippet.
I've been planning to include client version check into the gem binary to ensure that whoever is running it (Shipit/local 🎩 ) has the right version. We can do that later.